vlasovsolver/cpu_trans_map.cpp

```diff
@@ -654,66 +653,55 @@ void update_remote_mapping_contribution(
    int myRank;
    
    MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
+
```

Comment:
This could be in a timed barrier?

Reply:
That MPI_Comm_rank call is probably a debugging remnant, it's not used anywhere. Why do we want a barrier?

Reply:
No I mean the commented block after that.

Reply:
Well, that is also left overs for my debugging. Basically I was print out stuff from get_mpi_datatype() in spatialCell and I had to mark where the output from update_remote_mapping_contribution() starts and ends in order to parse the data with a script. I can do a round of cleanup of commented code if you want.

Reply:
As you like, I just looked at the diffs and commented on what made sense (or didn't quite) to me.